lib/conjur/cli.rb

```diff
@@ -91,6 +93,27 @@ def init!
         load_plugins
         commands_from 'conjur/command'
       end
+
+      def command_version_compatible? command
```

Comment:
The `appliance_version` helper seems OK, but I find the reworked `command_version_compatible?` harder to read.